.env.example

```
@@ -1,17 +1,17 @@
-OPENAI_API_KEY=sk-proj-...
-GEMINI_API_KEY=AIza...
-ANTHROPIC_API_KEY=sk-ant-...
-DEEPSEEK_API_KEY=sk-a...
-OPENROUTER_API_KEY=sk-or-v1-3...
-GROK_API_KEY=xai-US...
+OPENAI_API_KEY=<your-openai-api-key>
+GEMINI_API_KEY=<your-gemini-api-key>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GEMINI_API_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)